independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-codestart/java/runtime/src/main/codestarts/quarkus/{extension.id}-codestart/codestart.tpl.qute.yml

```diff
@@ -7,7 +7,7 @@ metadata:
   description: {#if extension.description}{extension.description}{#else}Start coding with {extension.name}{/if}
 #  path: /some-path
 {#if input.extra-codestarts.contains("quarkiverse")}
-  related-guide-section: https://quarkiverse.github.io/quarkiverse-docs/quarkus-quinoa/dev/index.html
+  related-guide-section: https://docs.quarkiverse.io/quarkus-quinoa/dev/index.html
```

Comment:
Why do we have quinoa hardcoded here?

Not sure what this `related-guide-section` metadata is about?

Reply:
Looks like it came in with https://github.com/quarkusio/quarkus/pull/26806. The [file was new in that PR](https://github.com/quarkusio/quarkus/blob/42f91e26a55a581739308e5325831f6a2a7c74de/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-codestart/java/runtime/src/main/codestarts/quarkus/%7Bextension.id%7D-codestart/codestart.tpl.qute.yml). 

I've raised https://github.com/quarkusio/quarkus/issues/43854 to track the discussion.